ExportScripts/export_standalone_monolithic.py

```diff
@@ -184,7 +185,7 @@ def process_command(args: list,
 
 # EXPORT SCRIPT STARTS HERE!
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(prog='Exporter for MultiplayerSample on windows',
+    parser = argparse.ArgumentParser(prog='Exporter for MultiplayerSample as a standalone release',
```

Comment:
I would avoid use of 'release' here to avoid confusion with the release build type. Perhaps "as a standalone build"?

Reply:
good idea, I can update the comment with that.